tools/IoTEdgeDevOps/DevOpsLib/BugWiqlQueries.cs

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+namespace DevOpsLib
+{
+    public static class BugWiqlQueries
+    {
+        public const string PrioritizedBugTemplate =
+@"SELECT
+    [System.Id],
+    [System.WorkItemType],
+    [System.Title],
+    [System.AssignedTo],
+    [System.State],
+    [System.Tags]
+FROM workitems
+WHERE
+    [System.TeamProject] = @project
+    AND [System.WorkItemType] = 'Bug'
+    AND [System.AreaPath] = 'One\IoT\Platform\{AREA}'
+    AND NOT [System.State] CONTAINS 'Done'
+    AND NOT [System.State] CONTAINS 'Removed'
+    AND [Microsoft.VSTS.Common.Priority] = {PRIORITY}";
+
+        public const string PrioritizedStartedBugTemplate =
+@"SELECT
+    [System.Id],
+    [System.WorkItemType],
+    [System.Title],
+    [System.AssignedTo],
+    [System.State],
+    [System.Tags]
+FROM workitems
+WHERE
+    [System.TeamProject] = @project
+    AND [System.WorkItemType] = 'Bug'
+    AND [System.AreaPath] = 'One\IoT\Platform\{AREA}'
+    AND NOT [System.State] CONTAINS 'Done'
+    AND NOT [System.State] CONTAINS 'Removed'
+    AND [Microsoft.VSTS.Common.Priority] = {PRIORITY}
+    AND [System.State] = 'In Progress'";
+
+        public const string UnprioritizedBugTemplate =
+@"SELECT
+    [System.Id],
+    [System.WorkItemType],
+    [System.Title],
+    [System.AssignedTo],
+    [System.State],
+    [System.Tags]
+FROM workitems
+WHERE
+    [System.TeamProject] = @project
+    AND [System.WorkItemType] = 'Bug'
+    AND [System.AreaPath] = 'One\IoT\Platform\{AREA}'
+    AND NOT [System.State] CONTAINS 'Done'
+    AND NOT [System.State] CONTAINS 'Removed'
+    AND NOT [Microsoft.VSTS.Common.Priority] IN (1)
```

Comment:
This is not modeled in a good way, but I think it is ok for now since this is an internal tool with time constraint

Reply:
Do you think this because of the existence of other possible bug states?

Reply:
No as this static class is to provide format of query; the actual value should not be in there; same as you will not create many of this query format for different area; which area should be pass-in.